app/views/kudos/show.html.erb

```diff
@@ -20,5 +20,10 @@
   <%= @kudo.giver_id %>
 </p>
 
-<%= link_to 'Edit', edit_kudo_path(@kudo) %> |
 <%= link_to 'Back', kudos_path %>
+<% if @kudo.giver == current_employee %>
+| <%= link_to 'Edit', edit_kudo_path(@kudo) %>
+<% end %>
+
+
```

Comment:
too many empty lines. It's a good practice to check a PR carefully before sending it to review, because otherwise others may think you're wasting their time or don't respect it